algviz/picture/connection_map.py

```diff
@@ -0,0 +1,136 @@
+import svgutils.transform as svgutils
+import os
+
+from algviz.parser import structures
+from .anchor import Anchor
+from .svg_engine import SVGEngine
+
+"""Tracks connection points and connection pictures to be composed into a single picture"""
```

Comment:
I think you're describing something I helped invent, but there's no way somebody unfamiliar would make head or tail of this comment